osu.Game/Screens/Play/PlayerSettings/PlayerSettingsDropdown.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Graphics;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Screens.Play.PlayerSettings
+{
+    public class PlayerSettingsDropdown<T> : OsuEnumDropdown<T>
+        where T : struct, Enum
+    {
+        protected override DropdownHeader CreateHeader() => new PlayerSettingsDropdownHeader();
+
+        protected override DropdownMenu CreateMenu() => new PlayerSettingsDropdownMenu();
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            AccentColour = colours.Yellow;
```

Comment:
The text doesn't look all that great. I think the text should change to black it it's active.

Can be done by deriving `OsuDropdownMenuItem` and setting `ForegroundSelected = Color4.Black` I believe.